AutonomousDemo/Velocity.cs

```diff
@@ -23,7 +23,6 @@ public Velocity(double speed = 0, double direction = Math.PI/2.0)
 
         public static Velocity operator +(Velocity vIntial, Velocity vFinal)
```

Comment:
First, don't use hungarian notation prefixes in C# (the 'v' in 'vInitial', for example). I already know they're velocities because I can see them being declared as such, and if I couldn't the IDE would still tell me that information.

Second, the latter argument shouldn't be "final" because it's clearly not the final velocity, since the final velocity is the sum of the two.